src/schema/__tests__/show.test.js

```diff
@@ -146,6 +146,27 @@ describe("Show type", () => {
     }
   })
 
+  it("returns a fair booth even with displayable set to true", async () => {
```

Comment:
Unless I'm missing something I think the test case should be for when `displayable == false`. 

(And fair is existy, as you've got here.)

So the example name and the fixture data need a little tweak here, right?

Reply:
Yes that's totally right. Thanks @anandaroop